### PR TITLE
Feature: email templating

### PR DESCRIPTION
**Pull request by ShellyXueHan**
Use EJS for email html templates

### CODE REVIEW

File: public/emailConfirmation.ejs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  table.button.google table td {
+    background: #DB4A39 !important;
+    border-color: #DB4A39;
+  }
+  .wrapper.secondary {
+    background: #f3f3f3;
+  }
+</style>
+
+<container>
```
Comment (patricksimonian) [inline public/emailConfirmation.ejs]: where are these xml tags coming from?

File: public/emailConfirmation.ejs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  </row>
+  <wrapper class="secondary">
+    <spacer size="16"></spacer>
+    <row>
+      <columns large="6">
+        <h5>BC Gov Reggie</h5>
+      </columns>
+    </row>
+  </wrapper>
+</container>
```
Comment (patricksimonian) [inline public/emailConfirmation.ejs]: new line

File: src/libs/email-utils.js [hunk 3]
```diff
@@ -41,21 +44,41 @@ export const setMailer = async (host, port) => {
...
+    // TODO: modify email contents and public host image/logo
+    const confirmLink = link ? 'https://www.google.ca' : 'https://www.google.com';
+    const logoLink = 'http://localhost:8000/gov-logo.png';
+    const htmlPayload = await ejs.renderFile('public/emailConfirmation.ejs', {
+      name: 'Reggie user',
+      confirmLink,
+      logoLink,
+    });
+
+    const textPayload = ejs.render(
```
Comment (patricksimonian) [inline src/libs/email-utils.js]: is this any different than creating a plain text version of an ejs file?

File: src/libs/email-utils.js [hunk 4]
```diff
@@ -41,21 +44,41 @@ export const setMailer = async (host, port) => {
   }
 };
 
-export const sendEmail = async (emailServerConfig, email) => {
+export const sendEmail = async (emailServerConfig, email, link) => {
```
Comment (patricksimonian) [inline src/libs/email-utils.js:53]: maybe add js docs on what each arg is for

**Review by patricksimonian — CHANGES_REQUESTED** [review]

File: src/libs/email-utils.js (hunk 3 above)
Comment (ShellyXueHan) [reply src/libs/email-utils.js]: no, moved to a .txt file in public

File: src/libs/email-utils.js (hunk 4 above)
Comment (ShellyXueHan) [reply src/libs/email-utils.js:53]: comments added in function

File: public/emailConfirmation.ejs (hunk 1 above)
Comment (ShellyXueHan) [reply public/emailConfirmation.ejs]: sample template...will modify those when updating content

Comment (ShellyXueHan) [reply public/emailConfirmation.ejs]: removed issue tags

**Pull request merged**